src/UI/Component/Glyph/Factory.php

```diff
@@ -499,4 +499,51 @@ public function note($action = null);
 	 * @return	\ILIAS\UI\Component\Glyph\Glyph
 	 */
 	public function comment($action = null);
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       Clicking the Like Glyph indicates a user affirms an item, e.g. a posting.
+	 *   composition: >
+	 *       The Like Glyph uses the "thumbs up" unicode emoji U+1F44D
```

Comment:
Add Period. Any hints about this "special unicode", any reference. Ist this some kind of standard. Maybe note, that this is not from the default bootstrap set, which is fine as long as we have a JF dec. for that.